core/embed/upymod/modtrezorio/modtrezorio.c

```diff
@@ -39,9 +39,10 @@
 
 uint32_t last_touch_sample_time = 0;
 
-#define CHECK_PARAM_RANGE(value, minimum, maximum)  \
-  if (value < minimum || value > maximum) {         \
-    mp_raise_ValueError(#value " is out of range"); \
+#define CHECK_PARAM_RANGE(value, minimum, maximum) \
+  if (value < minimum || value > maximum) {        \
+    const char *msg = (#value "out of range");     \
```

Comment:
missing space? ` out of range`

Reply:
Thanks! Fixed in https://github.com/trezor/trezor-firmware/pull/4961/commits/d26fb6a9726cb43f064580b4d1723767ebf1d258.